addons/attributes/initAttributes.sqf

```diff
@@ -185,6 +185,21 @@
     {alive _entity && {_entity isKindOf "LandVehicle" || {_entity isKindOf "Air"} || {_entity isKindOf "Ship"}}}
 ] call FUNC(addAttribute);
 
+[
+    "Object",
+    [LSTRING(BuildingMarker), LSTRING(BuildingMarker_Tooltip)],
+    QGVAR(icons),
+    [[
+        [false, "\a3\Ui_F_Curator\Data\default_ca.paa", "STR_sensoractiv_none", 14.5, 0.25, 2],
+        [true, "\a3\3den\Data\Displays\Display3DEN\PanelRight\submode_marker_area_ca.paa", LSTRING(BuildingMarker),  19.5, 0.25, 2]
+    ]],
+    {
+        [QEGVAR(common,setbuildingMarker), [SELECTED_OBJECTS select {_x isKindOf "Building"}, _value]] call CBA_fnc_serverEvent;
+    },
+    {"" isNotEqualTo (_entity getVariable [QEGVAR(common,buildingMarker), ""])},
```

Comment:
Here and in other places: use `==`/`!=` for string comparison and put `getVariable` first.

Reply:
https://github.com/zen-mod/ZEN/pull/704/commits/0ac3c6a9e07a18dd15e2c022fe5308b5576f73b9#diff-ab311763189315d2a6fea02999d8c7652230bab53f6b59959e3ac16ffdbfd10aR196
https://github.com/zen-mod/ZEN/pull/704/commits/df7388c87d42a6f409e1256b36478aa186d5e9f1#diff-1a0acf5a487bf5428782a6e3f09be3f440547e468684e97b9b9890ec5bfc40dbR23
https://github.com/zen-mod/ZEN/pull/704/commits/df7388c87d42a6f409e1256b36478aa186d5e9f1#diff-904878a4c3022a6347e5b85fe824f3641b2911718a3985159b1d6bdf0b3ad5e0R30